package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "4.7.1",
+  "version": "4.7.2",
```

Comment:
I'd personally bump the major version since this is a new feature, but no worries this works :)